src/migration/buildDatabase.ts

```diff
@@ -0,0 +1,147 @@
+import { createConnection, Connection } from 'mysql2';
```

Comment:
Is there a way we can modularize this for unit testing? That way we can setup/teardown the database during unit tests?
Refere to this for mongo instance https://github.com/edugator-cise/edugator-staff-backend/blob/main/src/config/database.ts

Reply:
Yes, definitely. We can do something precisely like that file, it may just be a bit shorter.